app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -96,7 +97,7 @@ def employer_poc
     # Dynamic Filter values for upcoming 30, 60, 90 days renewals
     @next_30_day = TimeKeeper.date_of_record.next_month.beginning_of_month
     @next_60_day = @next_30_day.next_month
```

Comment:
Do we even need to be defining these variables in the controller? The datatable seems to define its own.